src/java/org/apache/cassandra/config/DataRateSpec.java

```diff
@@ -296,7 +296,7 @@ public IntMebibytesPerSecondBound(long mebibytesPerSecond)
         // which were in megabits per second in 4.0. Do not start using it for any new properties
         public static IntMebibytesPerSecondBound megabitsPerSecondInMebibytesPerSecond(long megabitsPerSecond)
         {
-            final double MEBIBYTES_PER_MEGABIT = 0.119209289550781;
+            final double MEBIBYTES_PER_MEGABIT = 15625D / 131072D;
```

Comment:
I made these conversions more explicit, let me know if this is okay @ekaterinadimitrova2

Reply:
It is hard to see the reasoning behind the equations.

How about define the constants for MEGABIT, MEBIBYTES, etc. in bits and derive the conversion from the constants? For example,

```java
static final double MEBIBYTES_IN_BITS = Math.pow(2, 20) * 8;
static final double MEGABIT_IN_BITS = Math.pow(10, 6); 
static final double MEBIBYTES_PER_MEGABIT = MEGABIT_IN_BITS / MEBIBYTES_IN_BITS;
... more for the other size units...
```

Reply:
good point, I've updated the PR. The only difference is that I've defined `MEBIBYTES_IN_BITS` as:
```java
static final double MEBIBYTES_IN_BITS = Math.pow(2, 20) * 8;
```

Reply:
I am wondering what is the benefit from introducing these calculations? We can also add just a comment, at the same time the name already states - MEBIBYTES/MEGABIT.

Reply:
It is only to make the reasoning more clear since the procedure is coded. Adding a comment with the calculations also works. Meanwhile a few constants should not introduce any overkill.